### DISCUSSION TITLE
Hackage release

### DISCUSSION THREAD

**Discussion by siers** [AccelerateHS/accelerate#535] [General]
Will there be a new release on hackage with updated packages? I managed to build master with cabal dependencies from latest packges pinned in 23.05 nixpkgs.

**Comment by tmcdonell** [comment]
Hi @siers, welcome to the Accelerate project!

Short answer: yes, that would be great!
Long answer: no idea when I'll get time to do it. I'd like to finish my work on #534 and a couple other things I had planned, but, yeah maybe that isn't realistic.
